scripts/fix-node-pty-permissions.sh

```
@@ -0,0 +1,25 @@
+# "posix_spawnp failed." and the renderer never sees a useful error.
+#
+# Symptoms when this is broken: every pipeline thread fails immediately,
+# never produces a plan, and the issue card snaps to "failed" with no
+# visible error in the alert/dev console.
+set -e
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+
+# Find every spawn-helper inside node_modules (covers all archs + bun cache).
+find "$ROOT/node_modules" -type f -name spawn-helper -print0 2>/dev/null \
+	| while IFS= read -r -d '' helper; do
```

Shellcheck (0.11.0)

[warning] In POSIX sh, read -d is undefined.

(SC3045)